public/403.html

```diff
@@ -126,7 +126,7 @@ <h4 class="heading">Contact Us</h4>
             <h4 class="heading">Development</h4>
             <ul class="menu">
               <li><a href="/known_issues">Known Issues</a></li>
-              <li><a href="http://creativecommons.org/licenses/GPL/2.0/" title="View License">GPL/CC</a> by the <a href="http://transformativeworks.org/" title="The Organization for Transformative Works">OTW</a></li>
+              <li><a href="https://creativecommons.org/licenses/GPL/2.0/" title="View License">GPL/CC</a> by the <a href="https://transformativeworks.org/" title="The Organization for Transformative Works">OTW</a></li>
```

Comment:
Thanks for updating the error pages as well! Can you change the link to https://www.gnu.org/licenses/old-licenses/gpl-2.0.html, and the link text from "GPL/CC" to just "GPL"?

Reply:
Of course, I'll get to it this weekend!

Reply:
@redsummernight done!

Reply:
Can you also make this change in the other error pages?

Reply:
@redsummernight omg of course, I will get to it, sorry for the brain fart moment

Reply:
No worries, and no rush!